packages/client/src/contexts/ThemeContext.tsx

```diff
@@ -12,21 +14,48 @@ const defaultState = {
   isDarkTheme: false,
 };
 
-const localTheme = localStgMethodsObj.getValue('theme') as Theme;
-
 const ThemeContext = React.createContext<IThemeContext>(defaultState);
 
 export const ThemeContextProvider: React.FC<{
   children: React.ReactNode;
-}> = ({ children }) => {
+  cookies?: string | undefined;
+}> = ({ children, cookies }) => {
+  const localTheme = getCookie('theme', cookies) as Theme;
+
+  const { user } = userState();
+
   const [currentTheme, setCurrentTheme] = useState<Theme>(
     localTheme || Theme.LIGHT
   );
 
-  const handleThemeChange = () => {
+  useEffect(() => {
+    (async () => {
+      try {
+        if (!localTheme && user) {
+          const themeNameFromDB = await themeService.getUserTheme();
+          const themeNameFromDBIsValid =
+            typeof themeNameFromDB === 'string' &&
+            themeNameFromDB !== currentTheme &&
+            Object.values(Theme).includes(themeNameFromDB);
```

Comment:
Мне кажется достаточно так:
```suggestion
            Object.values(Theme).includes(themeNameFromDB) &&
            themeNameFromDB !== currentTheme;
```

Reply:
![image](https://user-images.githubusercontent.com/54811697/234291265-e02c1fba-d9c5-4e93-aee2-85118a737851.png)

Reply:
ок